packages/uiweb/src/lib/components/space/SpaceBanner/SpaceBanner.tsx

```diff
@@ -1,15 +1,264 @@
+  justify-content: center;
+  align-items: flex-start;
+
+}`;
+
+const HostName = styled.div`
+  display: flex;
+  flex-direction: row;
+  font-family: 'Strawford';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+
+}`;
+
+const Host = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 2px 8px;
+  margin-left: 8px;
+  line-height: 18px;
+
+  width: auto;
+  height: 19px;
+
+  background: rgba(255, 255, 255, 0.2);
+  border-radius: 6px;
+
+  font-family: "Strawford";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 10px;
+
+}`;
+
+const HostHandle = styled.div`
+  font-family: 'Strawford';
+  font-style: normal;
+
+}`;
+
+const Title = styled.div<{ orientation?: string }>`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  font-family: ${(props) =>
+    props.orientation === 'maximized' ? 'Strawford' : 'Strawford'};
+  font-weight: ${(props) =>
+    props.orientation === 'maximized' ? '700' : '500'};
+  font-size: ${(props) =>
+    props.orientation === 'maximized' ? '20px' : '12px'};
+  line-height: 130%;
+  width: 598px;
+  line-clamp: ${(props) => (props.orientation === 'maximized' ? '3' : '2')};
+
+}`;
+
+const Status = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  font-family: 'Strawford';
+  font-style: normal;
+
+}`;
+
+const Time = styled.div<{ orientation?: string }>`
+  display: ${(props) => (props.orientation === 'maximized' ? 'flex' : 'none')};
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+}`;
+
+const Icon = styled.img`
+  height: 24px;
+  width: 24px;
+  padding: 0 11px 0 0;
+  align-self: center;
+
+}`;
+
+const TimeText = styled.div`
+  font-family: 'Strawford';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 150%;
+
+}`;
+
+const Participants = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+
+}`;
+
+const ParticipantsIconContainer = styled.div<{ orientation?: string }>`
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  width: ${(props) => (props.orientation === 'maximized' ? '62px' : '46.5px')};
+  padding: 0 4px;
+
+}`;
+
+const ParticipantsIcon1 = styled.img` 
+  grid-column: 1 / 3;
+  grid-row: 1 / 2;
+  width: 31px;
+  height: 31px;
+  z-index: 3;
+
+}`;
+
+const ParticipantsIcon2 = styled.img`
+  grid-row: 1 / 2;
+  grid-column: 2 / -1;
+  width: 31px;
+  height: 31px;
+  z-index: 2;
+
+}`;
+
+const ParticipantsIcon3 = styled.img<{ orientation?: string }>`
+  display: ${(props) => (props.orientation === 'maximized' ? 'block' : 'none')};
+  grid-row: 1 / 2;
+  grid-column: 3 / -1;
+  width: 31px;
+  height: 31px;
+  z-index: 1;
+
+}`;
+
+const ParticipantsText = styled.div`
+  font-family: 'Strawford';
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 38px;
```

Comment:
dont use fixed width anywhere in the components. fixed widths break the frontend in unusual way in some screen sizes.

use auto width, or let the component inherit the width from its parent.

Fixed width should be used only in the case of media (images, videos, etc.)